content/webapp/views/components/RelatedWorksCard/helpers.tsx

```diff
@@ -3,28 +3,30 @@ import { WorkBasic } from '@weco/content/services/wellcome/catalogue/types';
 import { ContentApiLinkedWork } from '@weco/content/services/wellcome/content/types/api';
 
 export function transformCardData(work: WorkBasic | ContentApiLinkedWork) {
-  const isCatalogueWork = 'notes' in work;
+  // Type guard - check for catalogue-specific fields
```

Comment:
> The type guards in this file are definitely required for this branch's functionality - they allow the `RelatedWorksCard` component to work with both catalogue API works and content API works, which appears to be a feature of this branch.

I guess we'll have to decide whether we want to continue supporting the ability to display catalogue API works using RelatedWorksCards, or if we want to remove these guards when we remove the prototype